docs/docs/webchat/webchat-style.md

```diff
@@ -76,6 +76,8 @@ Here is a list of available styles. You need to modify them inside **src/webchat
 | `buttonStyle`           | Button styles. It can also be defined in `button.style`.                                                                                                                                      |
 | `buttonHoverBackground` | Background color when hovering over a button. It can also be defined in `button.hoverBackground`.                                                                                             |
 | `buttonHoverTextColor`  | Text color when hovering over a button. It can also be defined in `button.hoverTextColor`.                                                                                                    |
+| `buttonAutoDisable`    | Disables a button once the user has clicked on it. Default value:`true`. It can also be defined in `button.autodisable`. |
```

Comment:
same here, default value false?

Reply:
same here, default value false? @vanbasten17

Reply:
yes, the default value is false